### DISCUSSION TITLE
[fearure][windows] add topmost support

### DISCUSSION THREAD

**Discussion by congzhangzh** [doublecmd/doublecmd#1160] [Suggestions]
Hi folks,

Is that possible add topmost support on windows platform.

Do something like https://stackoverflow.com/questions/1528473/make-a-window-topmost-using-a-window-handle

Tks,
Cong

**Comment by alexx2000** [comment]
Hi,

Sorry I don't understand what do you mean.

**Comment by ags1234** [comment]
Do you mean you want to make Double Commander window always on top? 
[https://devblogs.microsoft.com/oldnewthing/20051121-10/?p=33263](url)

**Comment by congzhangzh** [comment]
> Do you mean you want to make Double Commander window always on top?
> [https://devblogs.microsoft.com/oldnewthing/20051121-10/?p=33263](url)

Yes

**Comment by congzhangzh** [comment]
> Hi,
> 
> Sorry I don't understand what do you mean.

Hi, I mean make the window always on top of the other application

**Comment by congzhangzh** [comment]
Something like https://ghisler.ch/board/viewtopic.php?t=28802

**Comment by alexx2000** [comment]
Really it is a strange feature, with such option all other applications will be invisible, so DC should be minimized to see them.

**Comment by WhiteChairFromIkea** [comment]
> Really it is a strange feature, with such option all other applications will be invisible, so DC should be minimized to see them.

"Always on top" feature is a neat thing to support. Notepad++ has it, realcalc also (replacement app for stock windows calculator), and I use thirty party system tray tool, like this one: https://github.com/navossoc/MenuTools, to make any window possible to be on top when I need this.

![1532h](https://user-images.githubusercontent.com/46539653/165039070-858c94a8-342d-49a2-b941-d9ae3b86ccfc.png)

**Comment by congzhangzh** [comment]
> Really it is a strange feature, with such option all other applications will be invisible, so DC should be minimized to see them.

Even windows’s taskmgr has an always on top option:)

**Comment by congzhangzh** [comment]
> Really it is a strange feature, with such option all other applications will be invisible, so DC should be minimized to see them.

Double cmd to be 1/4 screen is enough:)

**Comment by j2969719** [comment]
on linux this functionality for any application is usually provided by the window manager itself. not sure about windows, but in the days of winxp i remember that there was at least one third-party, though don't remember which one, program that also allowed you to pin an arbitrary application on top of other windows.

the examples mentioned above are utilities that take up a relatively small workspace when dc users quite often use it as a maximized window.
